third_party/gpus/cuda_configure.bzl

```diff
@@ -403,12 +427,15 @@ def _create_cuda_repository(repository_ctx):
        })
 
   # Set up crosstool/
+  cuda_include_path = _real_cuda_toolkit_path(repository_ctx, cuda_toolkit_path,
```

Comment:
If you are having this function return the CUDA toolkit path, perhaps combine it with `_cuda_toolkit_path()`?